util/design/data/multibits.h.tpl

```diff
@@ -0,0 +1,24 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Since this is template is used to autogenerate some code, we will probably want a warning message displayed here, which can be created using the `util/autogen_banner.py` module @sriyerg created in #8368. This could be created on the fly by importing and invoking functions form the `util/autogen_banner.py`module in `util/design/mubi/prim_mubi.py` below.

"WARNING this is autogen comment